Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -565,6 +590,12 @@ public class PBXProjGenerator {
 
         let excludedFiles: [String] = [".DS_Store"]
 
+        // Don't process the files of a bundle. Bundles are a special file type
+        // in Xcode ( not directories ).
+        if path.extension == "bundle" {
+            return ([getBundle(atPath: path)], [])
```

Comment:
Will anything break if we don't return the groups created by `getBundle` here, @yonaskolb ?